Automattic/wp-calypso
# NUX: Personalize copy based on the site type selection

#### Changes proposed in this Pull Request
* When the site type AB test is shown(10% users), the copy in the About step is personalized based on the selection made by the user in the site type step
* For the remaining users to whom the main flow (unchanged flow) is shown, the copy in the About step remains the same as it is now
* Listed below is the label shown in the About Step for site title and site topic respectively, based on the selection in the site type step

### Blog
What would you like to call your blog?
What will your blog be about?

### Business
What is the name of your business?
What type of business do you have?

### Professional
What is your name?
What type of work do you do?

### Education
What is the name of your site?
What will your site be about?


#### Testing instructions



Verify on the UI that for each site type selection, the copy in the About step reflects the segment.

## Steps
1. Activate the AB test:

```
localStorage.setItem( 'debug', 'calypso:abtests' );
localStorage.setItem( 'ABTests', '{"signupSegmentationStep_20181029":"include", "improvedOnboarding_20181023":"onboarding"}' );
```

2. Select site type as blog, then the About step should be:
<img width="486" alt="screenshot 2018-11-16 at 1 36 22 am" src="https://user-images.githubusercontent.com/1269602/48578583-0ec88080-e940-11e8-8216-735654034625.png">

3. Go back, select site type as Business, then professional and so on. Verify that the copy text matches the list given in the PR description.

4. Finally, deactivate the AB test and verify that in /start/about, the copy for the labels is as before. To deactivate the AB test:

```
localStorage.setItem( 'ABTests', '{"signupSegmentationStep_20181029":"exclude", "improvedOnboarding_20181023":"onboarding"}' );
```
